scripts/run.py

```diff
@@ -1409,14 +1413,17 @@ def check_clouds(self):
         """
         for all clouds, check if connection to h2o exists, and that h2o is healthy.
         """
-        time.sleep(3)
-        print("Checking cloud health...")
-        for c in self.clouds:
-            if self._h2o_exists_and_healthy(c.get_ip(), c.get_port()):
-                print("Node {} healthy.".format(c))
-            else:
-                print("Node with IP {} and port {} NOT HEALTHY" .format(c.get_ip(),c.get_port()))
-                # should an exception be thrown?
+        for i in range(5):
```

Comment:
Right. Then my question will be: WHY check the cloud? Makes no sense, if the tests have already passed. The effect actually is that the test is reported as failing. Let's not check anything at all at this moment. Either the tests failed, or they passed, that's it.